File: wxrtc/src/main/java/com/wx/rtc/bean/ProcessCommand.java
package com.wx.rtc.bean;

/**
 * @author wjf
 */
public @interface ProcessCommand {
    String START_ASR = "start_asr";
    String END_ASR = "end_asr";
    String END_AND_START_ASR = "end_and_start_asr";
    String START_QRCODE = "start_qrcode";
    String END_QRCODE = "end_qrcode";
    String START_BARCODE = "start_barcode";
    String END_BARCODE = "end_barcode";
    String START_THERMOMETER = "start_thermometer";
    String END_THERMOMETER = "end_thermometer";
    String START_DIGITAL_THERMOMETER = "start_digital_thermometer";
    String END_DIGITAL_THERMOMETER = "end_digital_thermometer";
    String START_INFRARED_THERMOMETER = "start_infrared_thermometer";
    String END_INFRARED_THERMOMETER = "end_infrared_thermometer";
    String START_SPHYGMOMANOMETER = "start_sphygmomanometer";
    String END_SPHYGMOMANOMETER = "end_sphygmomanometer";
    String START_DROP = "start_drop";
    String END_DROP = "end_drop";
}

File: wxrtc/src/commonMain/kotlin/com/wx/rtc/bean/ProcessCommand.kt
package com.wx.rtc.bean

/**
 * @author wjf
 */
internal object ProcessCommand {
    const val START_ASR: String = "start_asr"
    const val END_ASR: String = "end_asr"
    const val END_AND_START_ASR: String = "end_and_start_asr"
    const val START_QRCODE: String = "start_qrcode"
    const val END_QRCODE: String = "end_qrcode"
    const val START_BARCODE: String = "start_barcode"
    const val END_BARCODE: String = "end_barcode"
    const val START_THERMOMETER: String = "start_thermometer"
    const val END_THERMOMETER: String = "end_thermometer"
    const val START_DIGITAL_THERMOMETER: String = "start_digital_thermometer"
    const val END_DIGITAL_THERMOMETER: String = "end_digital_thermometer"
    const val START_INFRARED_THERMOMETER: String = "start_infrared_thermometer"
    const val END_INFRARED_THERMOMETER: String = "end_infrared_thermometer"
    const val START_SPHYGMOMANOMETER: String = "start_sphygmomanometer"
    const val END_SPHYGMOMANOMETER: String = "end_sphygmomanometer"
    const val START_DROP: String = "start_drop"
    const val END_DROP: String = "end_drop"
}
